tests/kani/Intrinsics/Math/powf64.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+pub fn f(a: u64) -> u64 {
+    const C: f64 = 0.618;
+    (a as f64).powf(C) as u64
+}
+
+#[cfg(kani)]
+mod verification {
+    use super::*;
+
+    #[kani::proof]
+    fn verify_f() {
+        const LIMIT: u64 = 10;
+        let x: u64 = kani::any();
+        let y: u64 = kani::any();
+        // outside these limits our approximation may yield spurious results
+        kani::assume(x > LIMIT && x < LIMIT * 3);
+        kani::assume(y > LIMIT && y < LIMIT * 3);
```

Comment:
These assumptions are different to the ones in the test from #2966 - how did you find them?

Reply:
Trial an error. (But also: the assumptions in the original test seemed strange for they effectively looked for cases where numeric error was likely.)